cookbooks/bcpc-hadoop/recipes/resource_manager.rb

```diff
@@ -85,6 +85,7 @@
   subscribes :restart, "template[/etc/hadoop/conf/yarn-env.sh]", :delayed
   subscribes :restart, "template[/etc/hadoop/conf/yarn-site.xml]", :delayed
   subscribes :restart, "template[/etc/hadoop/conf/mapred-site.xml]", :delayed
+  subscribes :restart, "template[/etc/hadoop/conf/container-executor.cfg]", :delayed
```

Comment:
Do all three services need to be restarted or just job history server and resource manager.